onnxruntime/test/testdata/transform/fusion/bias_dropout_residual_gen.py

```diff
@@ -0,0 +1,114 @@
+import onnx
```

Comment:
do we need to add copyright?

Reply:
didn't see copyright in other similar files... 
I guess they are be skipped.

---
In reply to: [441097640](https://github.com/microsoft/onnxruntime/pull/4167#discussion_r441097640) [](ancestors = 441097640)